crates/nu-command/src/conversions/into/duration.rs

```diff
@@ -122,6 +144,38 @@ impl Command for SubCommand {
                     span,
                 }),
             },
+            Example {
+                description: "Convert duration to the requested duration as a string",
+                example: "420sec | into duration --convert ms",
+                result: Some(Value::String {
```

Comment:
We need to have `into duration` return a duration, not a string. This was part of the reason for the move to `format duration`. Same comment on the other Examples that output a `Value::String`.